AmplifyPlugins/Predictions/AWSPredictionsPlugin/Support/Utils/IdentifyTextResultTransformers.swift

```diff
@@ -10,6 +10,7 @@ import AWSRekognition
 import Amplify
 import AWSTextract
 
+// swiftlint:disable type_body_length
```

Comment:
If you want to avoid this, you can create a file like:

`IdentifyTextResultTransformers+Tables.swift` (and similarly with key value)

and then put:
```
extension IdentifyTextResultTransformers {
    static func processTables(...
    static func processTable(_ tableBlock: AWSText...
    static func constructTableCell(_....
}
```